vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go

```diff
@@ -289,7 +289,10 @@ func SetFileLabel(fpath string, label string) error {
 	if fpath == "" {
 		return ErrEmptyPath
 	}
-	return lsetxattr(fpath, xattrNameSelinux, []byte(label), 0)
+	if err := lsetxattr(fpath, xattrNameSelinux, []byte(label), 0); err != nil {
+		return errors.Wrapf(err, "failed to set file label on %s", fpath)
+	}
+	return nil
```

Comment:
This change in selinux-go seems to cause a regression on our side (introduced in https://github.com/opencontainers/selinux/pull/60)

Reply:
@rhatdan do you have any idea how to fix the SELinux related issue?
```
failed to set file label on /var/lib/containers/storage/overlay/ed4a2284370c886d47ef5a6b0a183a7007b67a354a1f1c29fe478eb8e4399a0a/merged: operation not supported
```

Reply:
I am also hitting it -> https://github.com/cri-o/cri-o/pull/3205

Reply:
Looks like we should ignore ENOSUP errors.

Reply:
Was CRI-O examining the error, and checking for ENOSUP?  It needs to use errors.Cause now.

Reply:
Where is this being called?

Reply:
https://github.com/cri-o/cri-o/blob/master/server/label_linux.go#L11, I think.

Reply:
I was thinking the same but was not sure if it is an appropriate solution to skip those errors. What is the reason behind it?

I can spin up a follow-up PR tomorrow if nobody catches up first. :)

Reply:
In a number of places, we're already comparing the error we get back to syscall.ENOTSUP.  The change would be to pass those errors to `errors.Cause()` to unwrap them before comparing them.

Reply:
I believe this is the correct fix.

Reply:
Opened #3212.  If you prefer to cherry-pick it into this one and close that one, I'm fine with that.